cl/people_db/management/constants.py

```diff
@@ -0,0 +1,193 @@
+DISTRICT_COURTS = {
```

Comment:
I think I'd give these a name that reflects the data source they should be used with. Maybe `FJC_DISTRICT_COURTS`? It's worth putting an explanatory comment about the source in here too. Think of 2025-Mike, please.